File: Hito1/CampeonatoDeFutbol/src/main/java/danieldeveloper/campeonatodefutbol/Campeonato.java
package danieldeveloper.campeonatodefutbol;

public class Campeonato {
    private String nombreCampeonato;
    private Equipo[] equipos;
    
    public Campeonato(String name, Equipo[] team){
        this.nombreCampeonato = name;
        this.equipos = team;
    }
    
    public String getNombreCampeonato(){ return nombreCampeonato; }
    public void setNombreCampeonato(String nombreCampeonato) { this.nombreCampeonato = nombreCampeonato; }
    public Equipo[] getEquipo() { return equipos; }
    public void setEquipo(Equipo[] equipos){ this.equipos = equipos; }
    
    public void imprimir(){
        System.out.println("CAMPEONATO - " + this.nombreCampeonato);
        System.out.println("--------------------------------------");
        for(int i = 0; i < equipos.length; i++){
            equipos[i].imprimir();
            System.out.println("--------------------------------------");
        }
    }
}

File: Hito1/CampeonatoDeFutbolPHP/Campeonato.php
<?php
include "./Equipo.php";
	class Campeonato
	{
		private $nombreCampeonato = "";
		private $equipos[] = new Equipo;

		function __construct($name, $teams[])
		{
			$this->nombreCampeonato($name);
			$this->equipos($teams);
		}

		function getNombreCampeonato()
		{
			return $nombreCampeonato;
		}

		function setNombreCampeonato($name){
			$this->nombreCampeonato($name);
		}

		function getEquipos()
		{
			return $equipos;
		}

		function setEquipos($teams[] = new Equipo){
			$this->equipos($teams);
		}

		function imprimir(){
			echo $nombreCampeonato . "<br>";

			for ($i = 0; $i < count($equipos); $i++) { 
				$equipos[$i]->imprimir();
			}
		}
	}
?>